cmd/key-helper/importKey.go

```diff
@@ -140,7 +145,89 @@ func wrapPrivateKey(wrappingKey string, privateKeyPath string) (string, error) {
 	return base64Ciphertext, nil
 }
 
-func importKeyToTransit(ctx context.Context, client *vault.Client, ciphertext string, keyName string) error {
+// Checks PrivateKey type and bitlength in EC, PKCS1 and PKCS8 containers
```

Comment:
Personally, I expected changes above in `wrapPrivateKey`, and this code can largely be merged in there, IMO, conditioned off the type of the read key:

```go
switch typedKey := privateKey.(type) {
case *rsa.PrivateKey:
    ... &c
case ... &c
}
```

Then the returned key type can be merged alongside the `base64Ciphertext` value and not need to re-read the source key. My 2c.